content/en/references/docker-images.md

```diff
@@ -0,0 +1,46 @@
+
+You can use the Community image to start your LocalStack container using various [installation methods](https://docs.localstack.cloud/getting-started/installation/). While configuring to run LocalStack with Docker or Docker Compose, run the `localstack/localstack` image with the appropriate tag you have pulled (if not `latest`).
+
+## LocalStack Pro image
+
+LocalStack Pro contains various advanced extensions to the LocalStack base platform. With LocalStack Pro image, you can access all the emulated AWS cloud services running entirely on your local machine. To use the LocalStack Pro image, you can pull the image from Docker Hub:
+
+{{< command >}}
+$ docker pull localstack/localstack-pro:latest
+{{< / command >}}
+
+To use the LocalStack Pro image, you must configure an environment variable named `LOCALSTACK_API_KEY` to contain your API key. The LocalStack Pro image will display a warning if you do not set an API key (or the key is inactive) and will not activate the Pro features. LocalStack Pro gives you access to the complete set of LocalStack features, including [LocalStack Web Application](https://app.localstack.cloud) and [dedicated customer support](https://docs.localstack.cloud/getting-started/help-and-support/#pro-support). 
+
+You can use the Pro image to start your LocalStack container using various [installation methods](https://docs.localstack.cloud/getting-started/installation/). While configuring to run LocalStack with Docker or Docker Compose, run the `localstack/localstack-pro` image with the appropriate tag you have pulled (if not `latest`).
+
+{{< alert title="Notes" >}}
+Earlier, we maintained `localstack/localstack-light` and `localstack/localstack-full` images. They have been deprecated and are removed with the LocalStack 2.0 release. The [BigData image](https://hub.docker.com/r/localstack/bigdata/tags), which started as a `bigdata_container` container, has also been deprecated in favor of a BigData Mono container which installs dependencies directly into the LocalStack (`localstack_main`) container.
+{{< /alert >}}
+
+## Image tags
+
+We use tags for versions with significant features, enhancements, or bug fixes. To ensure that we move quickly and steadily, we run nightly builds, where all our updates are available on the `latest` tag of LocalStack's Docker image. We intend to announce more significant features and enhancements during major & minor releases. We will not create new releases for minor bug fixes and enhancements to ensure that we don't break your existing workflows.
```

Comment:
Not sure I follow the logic of the last sentence - can we slightly rephrase it to include a short description of patch releases? Something like:
```suggestion
We use tags for versions with significant features, enhancements, or bug fixes - following [semantic versioning](https://semver.org). To ensure that we move quickly and steadily, we run nightly builds, where all our updates are available on the `latest` tag of LocalStack's Docker image. We intend to announce more significant features and enhancements during major & minor releases. We occasionally create patch releases for minor bug fixes and enhancements, to ensure that we can deliver changes quickly while not breaking your existing workflows (in case you prefer not to use `latest`).
```